README.md

```diff
@@ -50,10 +38,16 @@ Full installation instructions are available in [INSTALL.md](https://github.com/
 ### Building static files
 
 We use Gulp as our JavaScript/CSS build system.
-node.js dependencies. Calling `gulp` on its own will build everything necessary
+node.js dependencies.
+After you started development versions of containers with `docker-compose`, connect to the main container:
+
+`docker-compose -f docker/docker-compose.dev.yml run webserver run /bin/bash`
+
+Calling `gulp` on its own will build everything necessary
 to access the server in a web browser:
 
-    ./node_modules/.bin/gulp
+    root@<container_id>:/code# npm install
+    root@<container_id>:/code# ./node_modules/.bin/gulp
```

Comment:
we need to work out a better way to do this - ideally we should install modules during Docker build, and then work out how to serve js and css during development and in production. I'm not sure how other MeB projects host static content, for my other projects I share a volume or host directory between my app container and nginx container and sync it during deploy